cfgov/jinja2/v1/_layouts/content-base-main-first.html

```diff
@@ -12,7 +12,10 @@
 {% block body_content scoped %}
 <div class="wrapper content_wrapper">
     <div class="content_main {% block content_main_modifiers -%}{%- endblock %}">
-        <section>
+        <section lang="{{ page.language }}"
```

Comment:
What are the benefits?

On Tue, Jan 26, 2016, 4:42 PM sebworks <notifications@github.com> wrote:

> In cfgov/jinja2/v1/_layouts/content-base-main-first.html
> <https://github.com/cfpb/cfgov-refresh/pull/1372#discussion_r50904930>:
>
> > @@ -12,7 +12,10 @@
> >  {% block body_content scoped %}
> >  <div class="wrapper content_wrapper">
> >      <div class="content_main {% block content_main_modifiers -%}{%- endblock %}">
> > -        <section>
> > +        <section lang="{{ page.language }}"
>
> Can we make this whole section a block or macro?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/cfgov-refresh/pull/1372/files#r50904930>.
>